fe1-web/package.json

```diff
@@ -39,7 +40,9 @@
     "expo": "^42.0.0",
     "expo-status-bar": "~1.0.4",
     "idb-keyval": "^5.0.4",
+    "jest-expo": "^43.0.1",
     "js-sha256": "^0.9.0",
+    "madge": "^5.0.1",
```

Comment:
I believe these (except `js-sha256`) should also be devDependencies.
By the way, I didn't know about madge - that's a cool tool !

Reply:
I forgot to remove madge after checking circular dependency because of the problem with the import order.

Reply:
It's a cool tool though !
Do you think it'd be useful to add it as a check in the CI ?

Reply:
It might not be useful to add it in the CI, because if there's a circular dependency, the tests will fail, we won't do a PR. But it will be useful to add it somewhere else like useful tools, to tell others what they can do in case of a similar problem.